libs/blocks/global-navigation/global-navigation.css

```diff
@@ -62,7 +62,7 @@ header.global-navigation {
   right: 20px; /* hamburger menu gutter */
   display: none;
   flex-direction: column;
-  height: 100vh;
+  height: calc(100vh - var(--feds-height-nav));
```

Comment:
There might be promos or other elements on top of the gnav, there's padding logic that should be handling this. Please add some screenshots and some valid test URLs so we can better understand what you're trying to fix.